manifests/m/MacroDeck/MacroDeck/2.12.0/MacroDeck.MacroDeck.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-22572-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: MacroDeck.MacroDeck
+PackageVersion: 2.12.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.1.0
```

Comment:
The latest manifest version is 1.5.0. This doesn't affect the PR, but I suggest updating your local branch and pulling in the latest changes to master so you get the latest updates to YamlCreate